.github/workflows/build-admb-and-ss3-from-source.yml

```diff
@@ -0,0 +1,290 @@
+# Builds the stock synthesis executables for testing and distribution.
+# for mac, linux, windows
+# Runs on every push and PR (even draft PRs)
+name: build-admb-and-ss3-from-source
+
+on:
+    workflow_dispatch:
+
```

Comment:
Could we add a schedule here so that it runs weekly or monthly so we figure out if anything breaks before we need to rely on it?
```  
schedule:
    - cron: '0 15 * * 2' // every Tuesday at 3pm UTC (= 10am or 11am Eastern time)
```
I think that is right based on syntax found here:https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule)